2. Web API now calls Microsoft Graph/TodoListService/TodoListService.csproj

```diff
@@ -11,8 +11,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Identity.Web" Version="1.8.0" />
-    <PackageReference Include="Microsoft.Identity.Web.MicrosoftGraph" Version="1.8.0" />
+    <PackageReference Include="Microsoft.Identity.Web" Version="1.9.0" />
```

Comment:
the reference to "Microsoft.Identity.Web" should not be needed (as it will be brought by "Microsoft.Identity.Web.MicrosoftGraph"